core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -43,7 +43,7 @@ oppia.directive('questionsList', [
       restrict: 'E',
       scope: {},
       bindToController: {
-        getTopicId: '&topicId',
+        isInTopicEditor: '&isInTopicEditor',
```

Comment:
Hi @sophiewu6, I don't think the questions-list directive should refer to the topic editor (or the page it is in) at all.

What actually needs to differ? Can you reframe as something like showSkillDescriptions perhaps -- something that is described in relation to the questions-list directive, not its surroundings? If you need multiple boolean variables, that is fine.

Reply:
Hmm yeah I have thought about that, but there are two things that needs to differ - skill descriptions and select skill modal. Is showSkillDescriptionsAndSelectionModal good enough?

Reply:
You can have two booleans.

Reply:
Okay cool I didn't think about that. Fixed it just now. PTAL!